.goreleaser.yml

```diff
@@ -0,0 +1,30 @@
+project_name: m3db
+builds:
+  - main: ./src/cmd/services/m3dbnode/main/main.go
+    binary: m3dbnode
+    ldflags: >-
+      -X {{.Env.BASE_PACKAGE}}.Revision={{.Env.GIT_REVISION}}
+      -X {{.Env.BASE_PACKAGE}}.Branch={{.Env.GIT_BRANCH}}
+      -X {{.Env.BASE_PACKAGE}}.Version={{.Env.GIT_VERSION}}
+      -X {{.Env.BASE_PACKAGE}}.BuildDate={{.Env.BUILD_DATE}}
+      -X {{.Env.BASE_PACKAGE}}.BuildTimeUnix={{.Env.BUILD_TS_UNIX}}
+      -X {{.Env.BASE_PACKAGE}}.LogBuildInfoAtStartup=true
+    goos:
+      - linux
+      - darwin
+      # Issue #692 tracks Windows support.
+      # - windows
+    goarch:
+      - amd64
+archive:
+  wrap_in_directory: true
+  format_overrides:
+    - goos: windows
+      format: zip
+  files:
```

Comment:
question for you - wanna include a config file or nah?

Reply:
A config? Not sure what you mean

Reply:
hm are you creating a zip or a binary at the end of this? my point was it might be easier for users if you had a zip with binary+sample config

Reply:
oh, gotcha. tbh I think it would be better to leave that as a tbd since I'm not sure what the story will be for open source deployment and I'm not sure how widely applicable the configs. If we get to a point where we only need to point m3db at a standard config to start it then I definitely think we should include one